member_organizations.yaml

```diff
@@ -0,0 +1,27 @@
+# This file provides information about companies/organizations
+# that would like to be aknowledged as contributing/participating in
+# Kubeflow.
+#
+# Required fields:
+#   name - The name of the company or organization.
+#   contacts - A list of email addresses for the primary contacts for
+#     the organization.
+#
+# Optional fields
+#    url - Url for the organization 
+#    hiringForKubeflow - true/false whether the orgniazation is hiring for Kubeflow related
+#      positions
+#    hiringUrl - A URL for potential job applicants.
+#
+# We do not infer organizations based on affiliation in members.yaml because
+# some members may be contributing as individuals and not part of a deeper involvement
+# with Kubeflow for their company/organization.
```

Comment:
I'd add note that this document is entirely voluntary and companies might contribute as they like without adding footprint there. A lot of companies just contributes one or two patches total and, while being very valid contributions, are reluctant to make commitments

Reply:
Isn't that covered by the first line of the file? "This file provides information about companies that would like to be acknowledged"
So if companies don't want to be acknowledged then they shouldn't add themselves.